tests/test_train_eval_pipeline.py

```diff
@@ -25,7 +25,7 @@ def setUp(self) -> None:
         self.data = {
             "datapath": self.testdata_mrc,
             "datatype": "mrc",
-            "limit": 100,
```

Comment:
Is this also for removing the test failures on git ?

Reply:
i think the limit here is irrelevant because the data is smaller than 100 anyway

Reply:
ok , great.